library/consul

```diff
@@ -1,6 +1,7 @@
 # maintainer: James Phillips <james@hashicorp.com> (@slackpad)
 
-latest: git://github.com/hashicorp/docker-consul@470868df3885ad93f45a2c63c648bf119a544fa4 0.X
+latest: git://github.com/hashicorp/docker-consul@2c2873f9d619220d1eef0bc46ec78443f55a10b5 0.X
+0.7.1: git://github.com/hashicorp/docker-consul@2c2873f9d619220d1eef0bc46ec78443f55a10b5 0.X
 0.7.0: git://github.com/hashicorp/docker-consul@470868df3885ad93f45a2c63c648bf119a544fa4 0.X
 v0.7.0: git://github.com/hashicorp/docker-consul@470868df3885ad93f45a2c63c648bf119a544fa4 0.X
```

Comment:
> Tags that are removed from a library file do not get removed from the Docker Hub, so that old versions can continue to be available for use, but are not maintained by upstream or the maintainer of the official image.
> \- https://github.com/docker-library/official-images#library-definition-files

Can you make a second PR to drop `0.7.0` and `v0.7.0`? Since `0.7.1` is their update, I assume these images will no longer get updates from hashicorp.